packages/developer-portal/src/components/pages/app-detail/app-detail-v8.tsx

```diff
@@ -0,0 +1,169 @@
+          <AppContent appDetailState={appDetailState} />
+        </Section>
+      )
+  }
+}
+
+const AppDetailV8: React.FC<AppDetailProps> = () => {
+  const appDetailState = useSelector(selectAppDetailState)
+  const appDetailData = useSelector(selectAppDetailData)
+  const appIcon = appDetailData.media?.filter(({ type }) => type === 'icon')[MEDIA_INDEX.ICON]
+  const isLoadingAppDetail = useSelector(selectAppDetailLoading)
+  const isLoadingInstallations = useSelector(selectInstallationsListLoading)
+  const installationsData = useSelector(selectInstallationsListData)
+  const unfetch = !appDetailState?.data || !installationsData
+  const [tab, setTab] = useState<string>('details')
+  const location = useLocation()
+  const history = useHistory()
+  const { pathname } = location
+  const [submitAppModalVisible, setSubmitAppModalVisible] = useState<boolean>(false)
+
+  if (isLoadingAppDetail || isLoadingInstallations || unfetch) {
+    return <Loader label="Loading" fullPage />
+  }
+
+  return (
+    <ErrorBoundary>
+      <FlexContainer className={elWFull} isFlexInitial>
+        <SecondaryNavContainer>
+          <Title>Apps</Title>
+          <SecondaryNav className={elMb8}>
+            <SecondaryNavItem
+              onClick={navigate(history, Routes.APPS)}
+              active={pathname === Routes.APPS && !submitAppModalVisible}
+            >
+              My Apps
+            </SecondaryNavItem>
+            <SecondaryNavItem active={/^\/v8\/apps\/[a-z0-9-]/.test(pathname)}>App Details</SecondaryNavItem>
+            <SecondaryNavItem active={/^\/v8\/apps\/[a-z0-9-]\/edit/.test(pathname)}>Edit App</SecondaryNavItem>
+            <SecondaryNavItem onClick={onShowSubmitAppModal(setSubmitAppModalVisible)} active={submitAppModalVisible}>
+              Create New App
+            </SecondaryNavItem>
+          </SecondaryNav>
+          <Icon className={elMb3} icon="myAppsInfographic" iconSize="large" />
+          <Subtitle>Apps Documentation</Subtitle>
+          <SmallText hasGreyText>
+            This is the dashboard for your applications created using the Reapit Foundations platform. If you have not
+            created an app before or you need help, please take the time to view our getting started guide.
+          </SmallText>
+          <Button className={elMb3} intent="neutral" onClick={openNewPage(ExternalPages.developerPortalDocs)}>
+            View Docs
+          </Button>
+        </SecondaryNavContainer>
+        <PageContainer>
+          <FlexContainer isFlexRow isFlexAlignCenter>
+            {
+              <img
+                className={cx('image', 'is-96x96', elMr6)}
+                src={appIcon?.uri || ImagePlaceHolder}
+                alt={appDetailData.name}
+              />
+            }
+            <Title>{appDetailData.name}</Title>
+          </FlexContainer>
+          <Tabs
+            isFullWidth
+            name="app_tabs"
+            onChange={(event) =>
+              // @ts-ignore
```

Comment:
Why ts-ignore? Can you not do `event.target.value ?? 'DEFAULT_TAB` or similar?

Reply:
Can't seem to solve this. onChange gives a strange event where Target is of type 'T' and this generic type doesn't have value as a property